src/System.Diagnostics.TextWriterTraceListener/tests/System.Diagnostics.TextWriterTraceListener.Tests.csproj

```diff
@@ -11,4 +11,8 @@
     <Compile Include="CtorsStreamTests.cs" />
     <Compile Include="DelimiterWriteMethodTests.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
+    <Compile Include="ConsoleTraceListenerTests.cs" />
```

Comment:
Should the files end with "netcoreapp.cs"

Reply:
Although, we presumably DO want these tests to run against NETFX.

Reply:
That is correct. Then I'll need to add a netfx configuration.